tests/Bridge/Doctrine/Transport/ConnectionTest.php

```diff
@@ -259,7 +259,7 @@ public function testConnectionCannotInsertASingleTaskWithExistingIdentifier(): v
         $serializer = $this->createMock(SerializerInterface::class);
 
         $task = $this->createMock(TaskInterface::class);
-        $task->expects(self::once())->method('getName')->willReturn('foo');
+        $task->expects(self::exactly(2))->method('getName')->willReturn('foo');
```

Comment:
As we already perform assertions later, maybe we can use $task = new NullTask('foo');?